webapp/src/components/AccordianLayout.tsx

```diff
@@ -0,0 +1,51 @@
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Divider,
+  Typography,
+} from "@mui/material";
+import React from "react";
+import Description from "components/Description";
+
+type Props = {
+  name: string;
+  description: string;
+  docURL: string;
```

Comment:
Just to be rigorous, this is not a URL. It's just part of the path, which is part of the complete URL. I would suggest calling it `link` to stay consistent with the `Description`'s prop.
```suggestion
  link: string;
```